enacts/monthly/maproom.py

```diff
@@ -211,21 +215,21 @@ def clip(x):
     tile = clip(data)
 
     groups = tile.groupby('T.year')
-    if varobj['id'] == "rfe":
+    if var == "Rainfall":
```

Comment:
I haven't traced through the code to see if this is actually the case, but I have a feeling this might break when we translate menu items into another language, or rename menu items (e.g. "ENACTS rainfall"). Menu items have display names that are distinct from the id so that you can change the display name without changing the id. Make sure program logic uses the id, not the display name.

Reply:
Very valid concern, however, I have not touched the menu in layout that has only a value entry thus meaning that display name = id. So that "id" here defined in the config file has not been implemented to address that issue. Or it was the intention and didn't happen.
Maybe it would be fair to leave this alone in that PR and instead put up an Issue to address all display-vs-id concern throughout our current tools?

Reply:
ok.